bedrock/firefox/views.py

```diff
@@ -548,6 +548,8 @@ def get_template_names(self):
                 template = 'firefox/whatsnew/index.id.html'
         elif locale == 'zh-TW' and not version.startswith('64.'):
             template = 'firefox/whatsnew/index.zh-TW.html'
+        elif version.startswith('66.'):
+            template = 'firefox/whatsnew/whatsnew-fx66.html'
```

Comment:
Would be good to add a unit test for this template?

Reply:
excellent point! :P